### ISSUE TITLE
Null values should not be imported

### ISSUE THREAD

**Issue by dianashk** [pelias/openaddresses#102]
http://pelias.github.io/compare/#/v1/autocomplete%3Fsources=oa&text=null

```
1)	NULL Null Null Null, Newport, WA, USA
2)	NULL Null Null Null, WA, USA
3)	NULL Null Null Null, Ione, WA, USA
```

Connected to pelias/pelias#127

**Comment by dianashk** [comment]
Looks like `undefined` is a common street name, too

http://pelias.github.io/compare/#/v1/search%3Fsize=40&sources=oa&text=undefined

```
1)	undefined 123, Finland
 2)	undefined 211, Finland
 3)	undefined 180, Finland
 4)	undefined 2, Oulu, Finland
 5)	undefined 37, Finland
 6)	undefined 215, Finland
 7)	undefined 205, Finland
 8)	undefined 209, Finland
 9)	undefined 217, Finland
10)	undefined 225, Finland
```

**Comment by trescube** [comment]
The OA source data has "NULL" as the literal values for number and street.

**Comment by dianashk** [comment]
Resolved via pelias/openaddresses#103
